pkg/action/install.go

```diff
@@ -459,7 +460,13 @@ func (i *Install) performInstall(rel *release.Release, toBeAdopted kube.Resource
 	if len(toBeAdopted) == 0 && len(resources) > 0 {
 		_, err = i.cfg.KubeClient.Create(resources)
 	} else if len(resources) > 0 {
-		_, err = i.cfg.KubeClient.Update(toBeAdopted, resources, i.Force)
+		if i.TakeOwnership {
+			if kubeClient, ok := i.cfg.KubeClient.(kube.InterfaceThreeWayMerge); ok {
```

Comment:
nit: You could remove this check since there is `var _ InterfaceThreeWayMerge = (*Client)(nil)` check that ensures the client implements this API. If you want to keep this check, then return an error in an `else` block